hack/dind-cluster.sh

```diff
@@ -650,6 +671,17 @@ function refresh() {
   echo "  Copying new runtime to cluster '${cluster_id}'..."
   copy-runtime "${origin_root}" "${config_root}/"
 
+  if [[ -n "${local_registry}" ]]; then
+    local registry_ip=$(get-registry-ip)
+    ensure-local-registry "${origin_root}" "${config_root}" "${registry_ip}" "${local_registry_images[@]}"
```

Comment:
`local_registry_images` is a string, and the list of images must be passed to `ensure-local-registry` in a single positional parameter; why use `[@]`?